src/request/api.ts

```diff
@@ -0,0 +1,5 @@
+import instance from "./request"
+
+export const PhoneLoginApi = (password: number | undefined, phone: string | undefined) => instance.post(`/login/cellphone?password=${password}&phone=${phone}`)

```

Comment:
```suggestion
export const phoneLoginApi = (password: number, phone: string) => instance.post(`/login/cellphone?password=${password}&phone=${phone}`)

```
post 请求应该直接在 data 字段里传递参数，而不是拼接url